src/main/io/vtx_smartaudio.c

```diff
@@ -707,7 +707,7 @@ bool vtxSmartAudioInit(void)
     // the SA protocol instead requires pulldowns, and therefore uses SERIAL_BIDIR_PP_PD instead of SERIAL_BIDIR_PP
     const serialPortConfig_t *portConfig = findSerialPortConfig(FUNCTION_VTX_SMARTAUDIO);
     if (portConfig) {
-        portOptions_e portOptions = SERIAL_STOPBITS_2 | SERIAL_BIDIR | SERIAL_BIDIR_PP_PD;
+        portOptions_e portOptions = SERIAL_STOPBITS_2 | SERIAL_BIDIR | SERIAL_BIDIR_PP_PD | SERIAL_BIDIR_NOPULL;
```

Comment:
I don’t believe you need `SERIAL_BIDIR_PP_PD`. See https://github.com/betaflight/betaflight/blob/2ff71b0a1f97ebc75f24bd73d0c4d63177d14f35/src/main/drivers/serial_softserial.c#L129

Reply:
You don't need it for softserial. But UART drivers use it to select GPIO_MODE_AF_PP / GPIO_MODE_AF_OD and smartaudio won't work with OD

Reply:
Do I need to drop commit 2 or is it okay @ledvinap

Reply:
I think that it won't work on UART without SERIAL_BIDIR_PP_PD. In this PR, I'd return it, to keep changes to minimum (UART driver does not test SERIAL_BIDIR_NOPULL)

Reply:
Agree - dropped commit 2 - this is a cherry-pick after all